stellar_account_prometheus_exporter/exporter.py

```diff
@@ -87,18 +95,18 @@ def do_GET(self):
                     self.error(504, 'Error retrieving data from {}'.format(url))
                     return
                 if not r.ok:
-                    self.error(504, 'Error retrieving data from {}'.format(url))
-                    return
-                if "balances" not in r.json():
-                    self.error(500, "Error - no balances found for account {}".format(account["account_id"]))
-                    return
-                if "subentry_count" not in r.json():
-                    self.error(500, "Error - no subentry_count found for account {}".format(account["account_id"]))
-                    return
-
-                labels = [network["name"], account["account_id"], account["account_name"]]
-                m_num_sponsored.labels(*labels).set(r.json().get("num_sponsored", 0))
-                m_num_sponsoring.labels(*labels).set(r.json().get("num_sponsoring", 0))
+                    m_account_exists.labels(*account_labels).set(0)
+                    logger.error("Error retrieving data from {}".format(url))
+                    continue
+                if "balances" not in r.json() or "subentry_count" not in r.json():
```

Comment:
Nice idea to combine it into one condition, makes code shorter :)

Reply:
:)